example/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="ext/codemirror.css" rel="stylesheet" type="text/css">
```

Comment:
In the notebook, they include the CodeMirror when they release it, but it is not in the repo.